README.md

```diff
@@ -42,8 +40,7 @@ args:
 
 ### `ref`
 
-A name of branch, git tag or a commit SHA to fetch. Will use remote default branch 
-(more precisly, remote HEAD) by default.
+Name of a branch, Git tag, or a commit SHA to fetch. Uses the default remote HEAD by default.
```

Comment:
```suggestion
Name of a branch, Git tag, or a commit SHA to fetch. Uses the remote HEAD by default.
```
There is only one HEAD on any remote. 

The point of the initial text was that not all developers have a good understanding of what HEAD is, and for them "default branch" will make more sense. But it isn't precise terminology.